libs/components/modals/src/lib/modules/modal/modal-host.component.ts

```diff
@@ -78,6 +85,17 @@ export class SkyModalHostComponent {
       provide: SkyModalConfiguration,
       useValue: params,
     });
+    params.providers.push({
```

Comment:
I removed this line and the demo seemed to work. Do we need to redefine the `SkyResizeObserverMediaQueryService` provider?

Reply:
There definitely needs to be a provider with `{ provide: SkyMediaQueryService, useExisting: SkyResizeObserverMediaQueryService }` so that there is a single instance per modal and `SkyResizeObserverMediaQueryService` stands in for the `SkyMediaQueryService`. This one is required for the demo to work.

Reply:
We talked offline; probably can get away with removing this line after all.